### PR TITLE
April UI maintenance: Accomodate common changes to push to /sdcard instead of /data/local/tmp

### PR DESCRIPTION
**Pull request by fadidurah**
April UI maintenance: Accomodate common changes to push to /sdcard instead of /data/local/tmp

As mentioned in the common pr (https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2029), we were running into issue when pushing our files to /data/local/tmp (firebase was not receiving them), so we have to instead push to /sdcard/, after which the files are copied into /data/local/tmp by our automation rules.

Also restored the promote to "release" step, previously we would only promote to "pre-release"

### CODE REVIEW

File: azure-pipelines/ui-automation/broker-release.yml [hunk 1]
```diff
@@ -599,15 +611,15 @@ stages:
...
-        otherFiles: "/data/local/tmp/CompanyPortal.apk=$(Pipeline.Workspace)/brokerapks/$(companyPortalApk),\
-                      /data/local/tmp/Authenticator.apk=$(Pipeline.Workspace)/brokerapks/$(authenticatorApk),\
-                      /data/local/tmp/OldAuthenticator.apk=$(Pipeline.WorkSpace)/brokerapks/oldAPKs/$(oldAuthenticatorApk),\
-                      /data/local/tmp/AzureSample.apk=$(Pipeline.Workspace)/AzureSample-RC/AzureSample-external-release.apk,\
-                      /data/local/tmp/BrokerHost.apk=$(Pipeline.WorkSpace)/BrokerHost-Dist-RC-release/brokerHost-dist-release.apk,\
-                      /data/local/tmp/Outlook.apk=$(Pipeline.WorkSpace)/firstpartyapks/$(outlookApk),\
-                      /data/local/tmp/Teams.apk=$(Pipeline.WorkSpace)/firstpartyapks/$(teamsApk),\
-                      /data/local/tmp/Word.apk=$(Pipeline.WorkSpace)/firstpartyapks/$(wordApk),\
-                      /data/local/tmp/Edge.apk=$(Pipeline.WorkSpace)/firstpartyapks/$(edgeApk)"
+        otherFiles: "/sdcard/CompanyPortal.apk=$(Pipeline.Workspace)/brokerapks/$(companyPortalApk),\
```
Comment (shahzaibj) [inline azure-pipelines/ui-automation/broker-release.yml:614]: Can we make this folder a parameter?

So we can easily change it back to `/data/local/tmp` in the future again if needed. (We don't necessarily have to make this paramterization change right now but just something to think about)

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-release.yml:614]: Yes, good call

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-release.yml:614]: I'll make it a variable tho

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-release.yml:614]: For the sake of time i'm going to tackle this post release

**Review by p3dr0rv — APPROVED** [review]

**Review by shahzaibj — APPROVED** [review]

**Pull request merged**